tools/devtools/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "yorkie-devtools",
+  "displayName": "Yorkie Devtools",
+  "version": "0.0.3",
```

Comment:
How about setting devtools version to be the same as JS SDK? Users can more easily imagine its compatible version with JS SDK.